ts/src/bingx.ts

```diff
@@ -1901,14 +1909,17 @@ export default class bingx extends Exchange {
         this.checkRequiredSymbol ('cancelOrders', symbol);
         await this.loadMarkets ();
         const market = this.market (symbol);
-        if (market['type'] !== 'swap') {
-            throw new BadRequest (this.id + ' cancelOrders is only supported for swap markets.');
-        }
         const request = {
             'symbol': market['id'],
-            'ids': ids,
         };
-        const response = await this.swapV2PrivateDeleteTradeBatchOrders (this.extend (request, params));
+        let response = undefined;
+        if (market['spot']) {
+            request['orderIds'] = ids;
```

Comment:
hum the signature fails in this endpoint
```
BadRequest bingx {"code":100400,"msg":"Key: 'OrdersCancelRequest.OrderIDs' Error:Field validation for 'OrderIDs' failed on the 'required' tag","debugMsg":""}
```

Reply:
It looks like the orderIds parameter is not so clear, we might need to check with them. I try both query string and post body, it didn't work.

```
$ node examples/js/cli bingx cancelOrders '["xxx","xxx", "xxx"]' ETH/USDT --verbose
2023-08-24T02:53:31.103Z
Node.js: v20.4.0
CCXT v4.0.72
bingx.cancelOrders (xxx, xxx, xxx, ETH/USDT)
fetch Request:
 bingx POST https://open-api.bingx.com/openApi/spot/v1/trade/cancelOrders?orderIds[]=xxx&orderIds[]=xxx&orderIds[]=xxx&symbol=ETH-USDT&timestamp=xxx&signature=xxx 
RequestHeaders:
 {
  'X-BX-APIKEY': 'xxx',
  'X-SOURCE-KEY': 'CCXT',
} 
RequestBody:
 

handleRestResponse:
 bingx POST bingx POST https://open-api.bingx.com/openApi/spot/v1/trade/cancelOrders?orderIds[]=xxx&orderIds[]=xxx&orderIds[]=xxx&symbol=ETH-USDT&timestamp=xxx&signature=xxx 200 OK 
ResponseHeaders:
 {
  ...
} 
ResponseBody:
 {"code":100400,"msg":"Key: 'OrdersCancelRequest.OrderIDs' Error:Field validation for 'OrderIDs' failed on the 'required' tag","debugMsg":""} 
```

```
$ node examples/js/cli bingx cancelOrders '["xxx","xxx", "xxx"]' ETH/USDT --verbose
2023-08-24T02:53:31.103Z
Node.js: v20.4.0
CCXT v4.0.72
bingx.cancelOrders (xxx, xxx, xxx, ETH/USDT)
fetch Request:
 bingx POST https://open-api.bingx.com/openApi/spot/v1/trade/cancelOrders? 
RequestHeaders:
 {
  'X-BX-APIKEY': 'xxx',
  'X-SOURCE-KEY': 'CCXT',
  'Content-Type': 'application/x-www-form-urlencoded'
} 
RequestBody:
 orderIds[0]=xxx&orderIds[1]=xxx&orderIds[2]=xxx&symbol=ETH-USDT&timestamp=xxx&signature=xxx

handleRestResponse:
 bingx POST https://open-api.bingx.com/openApi/spot/v1/trade/cancelOrders? 200 OK 
ResponseHeaders:
 {
  ...
} 
ResponseBody:
 {"code":100400,"msg":"Key: 'OrdersCancelRequest.OrderIDs' Error:Field validation for 'OrderIDs' failed on the 'required' tag","debugMsg":""} 

```